src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -130,18 +142,53 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
     }
   }
 
+  componentDidMount() {
+    if (
+      this.isArtaShipping() &&
+      !this.isCreateNewAddress() &&
+      !this.state.shippingQuoteId
+    ) {
+      this.selectShipping()
+    }
+  }
+
   // @ts-expect-error STRICT_NULL_CHECK
   getAddressList = () => this.props.me.addressConnection.edges
 
+  getOrderArtwork = () => this.props.order.lineItems?.edges?.[0]?.node?.artwork
+
   isCreateNewAddress = () => this.state.selectedSavedAddress === NEW_ADDRESS
 
+  isArtaShipping = () => {
```

Comment:
There might be an opportunity to add a lab feature check here!


The lab feature will allow us to gate this logic from being shown to users and only show it to admins/individual internal users before we launch. (and allow us to set `artaShippingEnabled=true` on some artworks for testing without exposing to users) Some additional documentation [here](https://github.com/artsy/gravity/blob/50aeb677c8cddc401161b65bffa09dba57167f82/doc/LabFeatures.md)

In Force, the check for a user's eligibility could look similar to [this check](https://github.com/artsy/force/pull/5579/files).

Curious to hear your thoughts, since we are working async, I have generated a lab feature in gravity staging called `"Collector Arta Shipping"` for admin users and added it to your artsymail user account in case you would like to use it.

Reply:
Yea, we discussed it with @ansor4, and if I remember correctly we decided to use **artaShippingEnabled** that comes from CMS as a lab feature key. I think if the ability to set **arta_enabled**(in CMS, `false` by default) will be hidden by the lab feature and collector won't have a chance to see the ARTA shipping quotes in Force. 

Also, I think it might break the page: for example, if we allow working with ARTA only for people with enabled lab feature and admin sets **arta_enabled** to **true** for an artwork what price should we show for the collectors without enabled lab feature? (according to the design there is no way to set shipping price if ARTA shipping is selected)